hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -468,8 +476,12 @@ private Partitioner getPartitioner(HoodieTable table, boolean isUpsert, Workload
     // cache writeStatusRDD before updating index, so that all actions before this are not triggered again for future
     // RDD actions that are performed after updating the index.
     writeStatusRDD = writeStatusRDD.persist(config.getWriteStatusStorageLevel());
+    indexTimer = metrics.getIndexLookupCtx();
     // Update the index back
     JavaRDD<WriteStatus> statuses = index.updateLocation(writeStatusRDD, jsc, table);
+    metrics.updateIndexMetrics("update",metrics.getDurationInMs(indexTimer == null ? 0L :
```

Comment:
small nit.. space after comma and "metrics."

Reply:
Done. Amends made